apps/src/aichat/views/aichatView.module.scss

```diff
@@ -11,7 +11,6 @@
   padding: 0;
   box-sizing: border-box;
   overflow: hidden;
-  user-select: none;
```

Comment:
Unrelated change to allow copying text (particularly messages). More discussion in [this Slack thread](https://codedotorg.slack.com/archives/C06UT1E2796/p1714149250477099).